auth_otp/views.py

```
@@ -0,0 +1,357 @@
+            user_agent=user_agent
+        )
+        
+        if success:
+            return Response(
+                {
+                    'success': True,
+                    'data': result
+                },
+                status=status.HTTP_201_CREATED
+            )
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)